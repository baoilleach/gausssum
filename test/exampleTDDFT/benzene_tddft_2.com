%chk=benzene.chk
%mem=12GB
%nproc=12
# RB3LYP/3-21G td iop(9/40=2) guess=read geom=allcheck
