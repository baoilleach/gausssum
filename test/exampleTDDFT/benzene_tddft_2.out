 Entering Gaussian System, Link 0=g09
 Initial command:
 /ichec/packages/gaussian/09b01/l1.exe /ichec/scratch/gaussian/Gau-14516.inp -scrdir=/ichec/scratch/gaussian/
 Entering Link 1 = /ichec/packages/gaussian/09b01/l1.exe PID=     14517.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003,2009,2010,
            Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 09 program.  It is based on
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 09, Revision B.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, B. Mennucci, 
 G. A. Petersson, H. Nakatsuji, M. Caricato, X. Li, H. P. Hratchian, 
 A. F. Izmaylov, J. Bloino, G. Zheng, J. L. Sonnenberg, M. Hada, 
 M. Ehara, K. Toyota, R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, 
 Y. Honda, O. Kitao, H. Nakai, T. Vreven, J. A. Montgomery, Jr., 
 J. E. Peralta, F. Ogliaro, M. Bearpark, J. J. Heyd, E. Brothers, 
 K. N. Kudin, V. N. Staroverov, T. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. Rendell, J. C. Burant, S. S. Iyengar, J. Tomasi, 
 M. Cossi, N. Rega, J. M. Millam, M. Klene, J. E. Knox, J. B. Cross, 
 V. Bakken, C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, 
 O. Yazyev, A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, 
 R. L. Martin, K. Morokuma, V. G. Zakrzewski, G. A. Voth, 
 P. Salvador, J. J. Dannenberg, S. Dapprich, A. D. Daniels, 
 O. Farkas, J. B. Foresman, J. V. Ortiz, J. Cioslowski, 
 and D. J. Fox, Gaussian, Inc., Wallingford CT, 2010.
 
 ******************************************
 Gaussian 09:  EM64L-G09RevB.01 12-Aug-2010
                 4-Nov-2012 
 ******************************************
 %chk=benzene.chk
 %mem=12GB
 %nproc=12
 Will use up to   12 processors via shared memory.
 ------------------------------------------------------
 # RB3LYP/3-21G td iop(9/40=2) guess=read geom=allcheck
 ------------------------------------------------------
 1/29=7,38=1/1;
 2/12=2,40=1/2;
 3/5=5,11=2,14=-4,16=1,25=1,30=1,74=-5,116=1/1,2,8,3;
 4/5=1/1;
 5/5=2,38=6/2;
 8/6=1,10=1/1;
 9/40=2,42=1/14;
 6/7=2,8=2,9=2,10=2/1;
 99/5=1,9=1/99;
 --------
 _Benzene
 --------
 Redundant internal coordinates taken from checkpoint file:
 benzene.chk
 Charge =  0 Multiplicity = 1
 H,0,0.,0.,0.0040897109
 C,0,0.,0.,1.0881082084
 C,0,1.2104365982,0.,1.7869541042
 H,0,2.1492241552,0.,1.2449448555
 C,0,1.2104365982,0.,3.1846458958
 H,0,2.1492241552,0.,3.7266551445
 C,0,0.,0.,3.8834917916
 H,0,0.,0.,4.9675102891
 C,0,-1.2104365982,0.,3.1846458958
 H,0,-2.1492241552,0.,3.7266551445
 C,0,-1.2104365982,0.,1.7869541042
 H,0,-2.1492241552,0.,1.2449448555
 Recover connectivity data from disk.
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          1           0        0.000000    0.000000    0.004090
      2          6           0        0.000000    0.000000    1.088108
      3          6           0        1.210437    0.000000    1.786954
      4          1           0        2.149224    0.000000    1.244945
      5          6           0        1.210437    0.000000    3.184646
      6          1           0        2.149224    0.000000    3.726655
      7          6           0        0.000000    0.000000    3.883492
      8          1           0        0.000000    0.000000    4.967510
      9          6           0       -1.210437    0.000000    3.184646
     10          1           0       -2.149224    0.000000    3.726655
     11          6           0       -1.210437    0.000000    1.786954
     12          1           0       -2.149224    0.000000    1.244945
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  H    0.000000
     2  C    1.084018   0.000000
     3  C    2.154939   1.397692   0.000000
     4  H    2.481710   2.154939   1.084018   0.000000
     5  C    3.403101   2.420873   1.397692   2.154939   0.000000
     6  H    4.298448   3.403101   2.154939   2.481710   1.084018
     7  C    3.879402   2.795384   2.420873   3.403101   1.397692
     8  H    4.963421   3.879402   3.403101   4.298448   2.154939
     9  C    3.403101   2.420873   2.795384   3.879402   2.420873
    10  H    4.298448   3.403101   3.879402   4.963421   3.403101
    11  C    2.154939   1.397692   2.420873   3.403101   2.795384
    12  H    2.481710   2.154939   3.403101   4.298448   3.879402
                    6          7          8          9         10
     6  H    0.000000
     7  C    2.154939   0.000000
     8  H    2.481710   1.084018   0.000000
     9  C    3.403101   1.397692   2.154939   0.000000
    10  H    4.298448   2.154939   2.481710   1.084018   0.000000
    11  C    3.879402   2.420873   3.403101   1.397692   2.154939
    12  H    4.963421   3.403101   4.298448   2.154939   2.481710
                   11         12
    11  C    0.000000
    12  H    1.084018   0.000000
 Stoichiometry    C6H6
 Framework group  D6H[3C2'(HC.CH)]
 Deg. of freedom     2
 Full point group                 D6H     NOp  24
 Largest Abelian subgroup         D2H     NOp   8
 Largest concise Abelian subgroup D2      NOp   4
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          1           0        0.000000    2.481710    0.000000
      2          6           0        0.000000    1.397692    0.000000
      3          6           0        1.210437    0.698846    0.000000
      4          1           0        2.149224    1.240855    0.000000
      5          6           0        1.210437   -0.698846    0.000000
      6          1           0        2.149224   -1.240855    0.000000
      7          6           0        0.000000   -1.397692    0.000000
      8          1           0        0.000000   -2.481710    0.000000
      9          6           0       -1.210437   -0.698846    0.000000
     10          1           0       -2.149224   -1.240855    0.000000
     11          6           0       -1.210437    0.698846    0.000000
     12          1           0       -2.149224    1.240855    0.000000
 ---------------------------------------------------------------------
 Rotational constants (GHZ):      5.6816867      5.6816867      2.8408434
 Standard basis: 3-21G (6D, 7F)
 There are    16 symmetry adapted basis functions of AG  symmetry.
 There are    11 symmetry adapted basis functions of B1G symmetry.
 There are     2 symmetry adapted basis functions of B2G symmetry.
 There are     4 symmetry adapted basis functions of B3G symmetry.
 There are     2 symmetry adapted basis functions of AU  symmetry.
 There are     4 symmetry adapted basis functions of B1U symmetry.
 There are    16 symmetry adapted basis functions of B2U symmetry.
 There are    11 symmetry adapted basis functions of B3U symmetry.
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
    66 basis functions,   108 primitive gaussians,    66 cartesian basis functions
    21 alpha electrons       21 beta electrons
       nuclear repulsion energy       203.1383991795 Hartrees.
 NAtoms=   12 NActive=   12 NUniq=    2 SFac= 4.00D+00 NAtFMM=   50 NAOKFM=F Big=F
 One-electron integrals computed using PRISM.
 NBasis=    66 RedAO= T  NBF=    16    11     2     4     2     4    16    11
 NBsUse=    66 1.00D-06 NBFU=    16    11     2     4     2     4    16    11
 Initial guess read from the checkpoint file:  benzene.chk
 B after Tr=     0.000000    0.000000    0.000000
         Rot=    1.000000    0.000000    0.000000    0.000000 Ang=   0.00 deg.
 Initial guess orbital symmetries:
       Occupied  (B1U) (E2G) (E2G) (E1U) (E1U) (A1G) (A1G) (E1U)
                 (E1U) (E2G) (E2G) (A1G) (B1U) (B2U) (E1U) (E1U)
                 (A2U) (E2G) (E2G) (E1G) (E1G)
       Virtual   (E2U) (E2U) (A1G) (E1U) (E1U) (B2G) (E2G) (E2G)
                 (B1U) (E2G) (E2G) (E1U) (E1U) (B1U) (A2G) (B2U)
                 (A1G) (A2U) (E2G) (E2G) (E1G) (E1G) (E1U) (E1U)
                 (E2U) (E2U) (A1G) (B2G) (B1U) (E2G) (E2G) (A2G)
                 (E1U) (E1U) (E1U) (E1U) (A1G) (B1U) (E2G) (E2G)
                 (E1U) (E1U) (E2G) (E2G) (B1U)
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 Keep R1 ints in memory in canonical form, NReq=3462520.
 SCF Done:  E(RB3LYP) =  -230.975772674     A.U. after    1 cycles
             Convg  =    0.1139D-10             -V/T =  2.0090
 ExpMin= 1.83D-01 ExpMax= 1.72D+02 ExpMxC= 1.72D+02 IAcc=1 IRadAn=         1 AccDes= 0.00D+00
 HarFok:  IExCor=  205 AccDes= 0.00D+00 IRadAn=         1 IDoV=-2
 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Range of M.O.s used for correlation:     7    66
 NBasis=    66 NAE=    21 NBE=    21 NFC=     6 NFV=     0
 NROrb=     60 NOA=    15 NOB=    15 NVA=    45 NVB=    45
 Orbital symmetries:
       Occupied  (B1U) (E2G) (E2G) (E1U) (E1U) (A1G) (A1G) (E1U)
                 (E1U) (E2G) (E2G) (A1G) (B1U) (B2U) (E1U) (E1U)
                 (A2U) (E2G) (E2G) (E1G) (E1G)
       Virtual   (E2U) (E2U) (A1G) (E1U) (E1U) (B2G) (E2G) (E2G)
                 (B1U) (E2G) (E2G) (E1U) (E1U) (B1U) (A2G) (B2U)
                 (A1G) (A2U) (E2G) (E2G) (E1G) (E1G) (E1U) (E1U)
                 (E2U) (E2U) (A1G) (B2G) (B1U) (E2G) (E2G) (A2G)
                 (E1U) (E1U) (E1U) (E1U) (A1G) (B1U) (E2G) (E2G)
                 (E1U) (E1U) (E2G) (E2G) (B1U)
    24 initial guesses have been made.
 Convergence on wavefunction:    0.001000000000000
 Iteration     1 Dimension    24 NMult    24
 CISAX will form    24 AO SS matrices at one time.
 New state      2 was old state      3
 No map to state      3
 Iteration     2 Dimension    30 NMult    30
 Iteration     3 Dimension    36 NMult    36
 Iteration     4 Dimension    42 NMult    42
 ***********************************************************************
 Excited states from <AA,BB:AA,BB> singles matrix:
 ***********************************************************************
 
 Ground to excited state transition densities written to RWF  633
 Ground to excited state transition electric dipole moments (Au):
       state          X           Y           Z        Dip. S.      Osc.
         1         0.0000      0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000      0.0000
 Ground to excited state transition velocity dipole moments (Au):
       state          X           Y           Z        Dip. S.      Osc.
         1         0.0000      0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000      0.0000
 Ground to excited state transition magnetic dipole moments (Au):
       state          X           Y           Z
         1         0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000
 Ground to excited state transition velocity quadrupole moments (Au):
       state          XX          YY          ZZ          XY          XZ          YZ
         1         0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000      0.0000      0.0000
 <0|del|b> * <b|rxdel|0> + <0|del|b> * <b|delr+rdel|0>
 Rotatory Strengths (R) in cgs (10**-40 erg-esu-cm/Gauss)
       state          XX          YY          ZZ    R(velocity)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000
 1/2[<0|r|b>*<b|rxdel|0> + (<0|rxdel|b>*<b|r|0>)*]
 Rotatory Strengths (R) in cgs (10**-40 erg-esu-cm/Gauss)
       state          XX          YY          ZZ     R(length)
         1         0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000
  1/2[<0|del|b>*<b|r|0> + (<0|r|b>*<b|del|0>)*] (Au)
       state          X           Y           Z        Dip. S.   Osc.(frdel)
         1         0.0000      0.0000      0.0000      0.0000      0.0000
         2         0.0000      0.0000      0.0000      0.0000      0.0000
         3         0.0000      0.0000      0.0000      0.0000      0.0000

 Excitation energies and oscillator strengths:
 
 Excited state symmetry could not be determined.
 Excited State   1:      Singlet-?Sym    5.6887 eV  217.95 nm  f=0.0000  <S**2>=0.000
      14 -> 24         0.01043
      20 -> 23         0.49991
      21 -> 22         0.49991
 This state for optimization and/or second-order correction.
 Total Energy, E(TD-HF/TD-KS) =  -230.766716834    
 Copying the excited state density for this state as the 1-particle RhoCI density.
 
 Excited state symmetry could not be determined.
 Excited State   2:      Singlet-?Sym    6.5709 eV  188.69 nm  f=0.0000  <S**2>=0.000
       7 -> 30        -0.01047
       7 -> 35         0.01110
       8 -> 29         0.01652
       9 -> 28        -0.01652
      10 -> 33         0.02346
      11 -> 34         0.02346
      12 -> 35        -0.01214
      13 -> 24        -0.01064
      14 -> 36         0.02844
      14 -> 53        -0.01140
      15 -> 31         0.01903
      15 -> 41         0.01175
      16 -> 32        -0.01903
      16 -> 40        -0.01175
      17 -> 27         0.03819
      18 -> 26         0.01617
      18 -> 34         0.01760
      18 -> 45         0.01604
      19 -> 25        -0.01617
      19 -> 33         0.01760
      19 -> 44        -0.01604
      20 -> 22        -0.49821
      21 -> 23         0.49821
      10 <- 33         0.01332
      11 <- 34         0.01332
      14 <- 36         0.01704
      17 <- 27         0.02472
      18 <- 45         0.01021
      19 <- 44        -0.01021
      20 <- 22         0.03009
      21 <- 23        -0.03009
 
 Excited state symmetry could not be determined.
 Excited State   3:      Singlet-?Sym    8.0842 eV  153.37 nm  f=0.0000  <S**2>=0.000
      10 -> 22         0.02251
      11 -> 23        -0.02251
      14 -> 27         0.04263
      18 -> 23        -0.49858
      19 -> 22         0.49859
 SavETr:  write IOETrn=   770 NScale= 10 NData=  16 NLR=1 LETran=      64.

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (B1U) (E2G) (E2G) (E1U) (E1U) (A1G) (A1G) (E1U)
                 (E1U) (E2G) (E2G) (A1G) (B1U) (B2U) (E1U) (E1U)
                 (A2U) (E2G) (E2G) (E1G) (E1G)
       Virtual   (E2U) (E2U) (A1G) (E1U) (E1U) (B2G) (E2G) (E2G)
                 (B1U) (E2G) (E2G) (E1U) (E1U) (B1U) (A2G) (B2U)
                 (A1G) (A2U) (E2G) (E2G) (E1G) (E1G) (E1U) (E1U)
                 (E2U) (E2U) (A1G) (B2G) (B1U) (E2G) (E2G) (A2G)
                 (E1U) (E1U) (E1U) (E1U) (A1G) (B1U) (E2G) (E2G)
                 (E1U) (E1U) (E2G) (E2G) (B1U)
 The electronic state is 1-A1G.
 Alpha  occ. eigenvalues --  -10.12841 -10.12777 -10.12777 -10.12663 -10.12663
 Alpha  occ. eigenvalues --  -10.12558  -0.85531  -0.74612  -0.74612  -0.60477
 Alpha  occ. eigenvalues --   -0.60477  -0.52831  -0.46078  -0.45260  -0.42612
 Alpha  occ. eigenvalues --   -0.42612  -0.36608  -0.34618  -0.34618  -0.25101
 Alpha  occ. eigenvalues --   -0.25101
 Alpha virt. eigenvalues --    0.00517   0.00517   0.10658   0.16437   0.16437
 Alpha virt. eigenvalues --    0.16533   0.20649   0.20649   0.21256   0.32441
 Alpha virt. eigenvalues --    0.32441   0.33698   0.33698   0.57640   0.57721
 Alpha virt. eigenvalues --    0.66749   0.68919   0.69599   0.73880   0.73880
 Alpha virt. eigenvalues --    0.78734   0.78734   0.80454   0.80454   0.83241
 Alpha virt. eigenvalues --    0.83241   0.88416   0.90632   0.97127   1.01027
 Alpha virt. eigenvalues --    1.01027   1.10876   1.11537   1.11537   1.30069
 Alpha virt. eigenvalues --    1.30069   1.32899   1.45189   1.46554   1.46554
 Alpha virt. eigenvalues --    1.54639   1.54639   1.88990   1.88990   2.47380
          Condensed to atoms (all electrons):
              1          2          3          4          5          6
     1  H    0.504745   0.390005  -0.040378  -0.003707   0.003824  -0.000051
     2  C    0.390005   5.139338   0.457890  -0.040378  -0.081385   0.003824
     3  C   -0.040378   0.457890   5.139338   0.390005   0.457890  -0.040378
     4  H   -0.003707  -0.040378   0.390005   0.504745  -0.040378  -0.003707
     5  C    0.003824  -0.081385   0.457890  -0.040378   5.139338   0.390005
     6  H   -0.000051   0.003824  -0.040378  -0.003707   0.390005   0.504745
     7  C   -0.000167  -0.023039  -0.081385   0.003824   0.457890  -0.040378
     8  H    0.000002  -0.000167   0.003824  -0.000051  -0.040378  -0.003707
     9  C    0.003824  -0.081385  -0.023039  -0.000167  -0.081385   0.003824
    10  H   -0.000051   0.003824  -0.000167   0.000002   0.003824  -0.000051
    11  C   -0.040378   0.457890  -0.081385   0.003824  -0.023039  -0.000167
    12  H   -0.003707  -0.040378   0.003824  -0.000051  -0.000167   0.000002
              7          8          9         10         11         12
     1  H   -0.000167   0.000002   0.003824  -0.000051  -0.040378  -0.003707
     2  C   -0.023039  -0.000167  -0.081385   0.003824   0.457890  -0.040378
     3  C   -0.081385   0.003824  -0.023039  -0.000167  -0.081385   0.003824
     4  H    0.003824  -0.000051  -0.000167   0.000002   0.003824  -0.000051
     5  C    0.457890  -0.040378  -0.081385   0.003824  -0.023039  -0.000167
     6  H   -0.040378  -0.003707   0.003824  -0.000051  -0.000167   0.000002
     7  C    5.139338   0.390005   0.457890  -0.040378  -0.081385   0.003824
     8  H    0.390005   0.504745  -0.040378  -0.003707   0.003824  -0.000051
     9  C    0.457890  -0.040378   5.139338   0.390005   0.457890  -0.040378
    10  H   -0.040378  -0.003707   0.390005   0.504745  -0.040378  -0.003707
    11  C   -0.081385   0.003824   0.457890  -0.040378   5.139338   0.390005
    12  H    0.003824  -0.000051  -0.040378  -0.003707   0.390005   0.504745
 Mulliken atomic charges:
              1
     1  H    0.186040
     2  C   -0.186040
     3  C   -0.186040
     4  H    0.186040
     5  C   -0.186040
     6  H    0.186040
     7  C   -0.186040
     8  H    0.186040
     9  C   -0.186040
    10  H    0.186040
    11  C   -0.186040
    12  H    0.186040
 Sum of Mulliken atomic charges =   0.00000
 Mulliken charges with hydrogens summed into heavy atoms:
              1
     2  C    0.000000
     3  C    0.000000
     5  C    0.000000
     7  C    0.000000
     9  C    0.000000
    11  C    0.000000
 Sum of Mulliken charges with hydrogens summed into heavy atoms =   0.00000
 Electronic spatial extent (au):  <R**2>=            459.1449
 Charge=              0.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.0000    Y=              0.0000    Z=              0.0000  Tot=              0.0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -31.1853   YY=            -31.1853   ZZ=            -39.9033
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=              2.9060   YY=              2.9060   ZZ=             -5.8120
   XY=              0.0000   XZ=              0.0000   YZ=              0.0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=              0.0000  YYY=              0.0000  ZZZ=              0.0000  XYY=              0.0000
  XXY=              0.0000  XXZ=              0.0000  XZZ=              0.0000  YZZ=              0.0000
  YYZ=              0.0000  XYZ=              0.0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=           -262.2584 YYYY=           -262.2584 ZZZZ=            -41.4569 XXXY=              0.0000
 XXXZ=              0.0000 YYYX=              0.0000 YYYZ=              0.0000 ZZZX=              0.0000
 ZZZY=              0.0000 XXYY=            -87.4195 XXZZ=            -61.2618 YYZZ=            -61.2618
 XXYZ=              0.0000 YYXZ=              0.0000 ZZXY=              0.0000
 N-N= 2.031383991795D+02 E-N=-9.406256406314D+02  KE= 2.289168767325D+02
 Symmetry AG   KE= 7.373855245202D+01
 Symmetry B1G  KE= 3.728381428702D+01
 Symmetry B2G  KE= 2.229160926733D+00
 Symmetry B3G  KE= 2.229160926733D+00
 Symmetry AU   KE= 3.424122837666D-16
 Symmetry B1U  KE= 1.863986945826D+00
 Symmetry B2U  KE= 7.144023112337D+01
 Symmetry B3U  KE= 4.013197007078D+01

 Test job not archived.
 1\1\GINC-R2I1N2\SP\RB3LYP TD-FC\3-21G\C6H6\NOBOYLE\04-Nov-2012\0\\# RB
 3LYP/3-21G td iop(9/40=2) guess=read geom=allcheck\\_Benzene\\0,1\H,0,
 0.,0.,0.0040897109\C,0,0.,0.,1.0881082084\C,0,1.2104365982,0.,1.786954
 1042\H,0,2.1492241552,0.,1.2449448555\C,0,1.2104365982,0.,3.1846458958
 \H,0,2.1492241552,0.,3.7266551445\C,0,0.,0.,3.8834917916\H,0,0.,0.,4.9
 675102891\C,0,-1.2104365982,0.,3.1846458958\H,0,-2.1492241552,0.,3.726
 6551445\C,0,-1.2104365982,0.,1.7869541042\H,0,-2.1492241552,0.,1.24494
 48555\\Version=EM64L-G09RevB.01\State=1-A1G\HF=-230.9757727\RMSD=1.139
 e-11\PG=D06H [3C2'(H1C1.C1H1)]\\@


 NOTHING PUZZLES ME MORE THAN TIME AND SPACE; AND YET NOTHING TROUBLES ME LESS
                                                          -- CHARLES LAMB (1810)
 Job cpu time:  0 days  0 hours  1 minutes 42.0 seconds.
 File lengths (MBytes):  RWF=     18 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 09 at Sun Nov  4 13:14:14 2012.
